src/convertToObject.js

```diff
@@ -11,8 +11,25 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+  const result = {};
+
+  for (let i = 0; i < sourceArray.length; i++) {
+    const element = sourceArray[i].split(':');
+
+    if (element[0] < 1 || element[1].length < 1) {
```

Comment:
`element[0] < 1` looks strange
element[0] is a string, so you check if the string is less than number 1?

Reply:
little modified